docs/src/snforge-advanced-features/profiling.md

```diff
@@ -12,6 +12,11 @@ All you have to do is use the [`--save-trace-data`](../appendix/snforge/test.md#
 $ snforge test --save-trace-data
 ```
 
+> 💡 **Tip**
+>
+> You can choose which resource to track (cairo-steps or sierra-gas) using `--tracked-resource` flag
+> Tracking sierra gas is only available for sierra 1.7.0+
+
```

Comment:
I assume this will (or is) be integrated in the profiler aftewards?

Reply:
Yup, I have already made all the necessary changes, just need to clean up and open up a PR (haven't look into it in two~three weeks now :O ). I can assign you to it once its opened, if you wish so :)